config/config.go

```diff
@@ -443,7 +443,7 @@ func DefaultRPCConfig() *RPCConfig {
 		CORSAllowedOrigins:     []string{},
 		CORSAllowedMethods:     []string{http.MethodHead, http.MethodGet, http.MethodPost},
 		CORSAllowedHeaders:     []string{"Origin", "Accept", "Content-Type", "X-Requested-With", "X-Server-Time"},
-		GRPCListenAddress:      "",
+		GRPCListenAddress:      "localhost:9090",
```

Comment:
We can't use 9090 here as that's the default app grpc. That will cause conflicts.
Pretty sure we also need protocol and port here.

e.g. `"tcp://0.0.0.0:9098"`

Reply:
could also set this here right? https://github.com/celestiaorg/celestia-app/blob/main/app/default_overrides.go#L255

Reply:
+1 we need the `tcp://` portion because https://github.com/celestiaorg/celestia-app/pull/4630#issuecomment-2822569122 

Agreed I can move this to celestia-app default overrides instead.

Out of curiosity, how was the port `:9098` picked? Or is that just an example?

Reply:
Yeah, its just an example. No reasoning there other than I don't think its used by anything else

Reply:
SGTM. Created https://github.com/celestiaorg/celestia-app/pull/4637